pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -90,6 +94,14 @@ const (
 	RunSpecStatusCancelled RunSpecStatus = "RunCancelled"
 )
 
+// RunSpecStatusMessage defines human readable status messages for the TaskRun.
+type RunSpecStatusMessage string
+
+const (
+	// Run cancelled as the Pipeline running it was cancelled.
+	RunCancelledByPipelineMsg RunSpecStatusMessage = "Run cancelled as pipeline has been canceled."
```

Comment:
Should be `pipeline has been cancelled`. =)

Reply:
Also, I think Pipeline-related constants should be defined either in the reconciler or in pipeline_types.

Reply:
Fixed the spelling to be cancelled.

Reply:
These constants are pipeline related but are set on a Run object and pipeline does not have a field with this object. 
As such these constants, I think, should be in run_types.go.